packages/story-editor/src/components/tabview/index.js

```diff
@@ -317,7 +317,12 @@ TabView.propTypes = {
   tabs: PropTypes.array.isRequired,
   tab: PropTypes.string.isRequired,
   tabRefs: PropTypes.objectOf(
-    PropTypes.shape({ current: PropTypes.instanceOf(Element) })
+    PropTypes.shape({
+      current:
+        typeof Element !== 'undefined'
+          ? PropTypes.instanceOf(Element)
+          : PropTypes.any,
+    }) // To handle SSR
```

Comment:
Does `PropTypes.elementType` work instead?

Reply:
It doesn't because `PropTypes.elementType` is for React Element.

<img width="563" alt="image" src="https://user-images.githubusercontent.com/6297436/163418744-1a0113ae-3f37-4692-ad40-45b6c0e2311f.png">